docs/introduction/actions.md

```diff
@@ -0,0 +1,24 @@
+Whenever the `state` object tree in your `store` needs to be updated in response to an event, you `dispatch` an `action` to your `store` with any relevant information required to make said update. An `action` must be a Lua table with a `type` field. They are usually created via an action creator Lua module like the following:
```

Comment:
Actions must be tables with a `type` field _unless you have middleware that will handle them_. Instead of mentioning middleware here, maybe we should just say that actions are _usually_ tables with a `type` field?